modules/xfem/src/base/XFEM.C

```diff
@@ -40,17 +38,14 @@ XFEM::XFEM(const InputParameters & params) : XFEMInterface(params), _efa_mesh(Mo
 
 XFEM::~XFEM()
 {
-  for (std::size_t i = 0; i < _geometric_cuts.size(); ++i)
-    delete _geometric_cuts[i];
-
   for (std::map<unique_id_type, XFEMCutElem *>::iterator cemit = _cut_elem_map.begin();
        cemit != _cut_elem_map.end();
        ++cemit)
     delete cemit->second;
```

Comment:
I spoke to Ben and Wen about this, and they said that while this is not an ideal solution, that this and several other naked new and delete instances would be addressed in a future code cleanup pull request for XFEM.